### DISCUSSION TITLE
Can't delete osd pod

### DISCUSSION THREAD

**Discussion by mc956759** [rook/rook#11680] [Q&A]
I want to remove an osd forever.
The process is:
1. set operator replicas to zero : kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=0
2. remove device by modifying cluster.yaml
3. remove osd by ceph command in rook-ceph-tools
4. delete pod:kubectl delete deploy -n rook-ceph rook-ceph-osd-0, and this step is ok
5. set operator replicas to 1: kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
But after step 5, I found that osd-pod-0 start again, and of course it can't running ok.
Why can't I delete pod osd-0? Any step I missed?
Thanks for any help!

**Comment by parth-gr** [comment]
4 delete pod:kubectl delete deploy -n rook-ceph rook-ceph-osd-0, and this step is ok
INstead try deleting the deployment of it

**Comment by travisn** [comment]
@mc956759 Did you see the [OSD removal instructions](https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-osd-mgmt/#remove-an-osd)?
